src/containers/intake/schemas/IntakeSchemas.js

```diff
@@ -51,6 +51,19 @@ const dataSchema = {
   title: '',
   properties: {
     [getPageSectionKey(1, 1)]: {
+      type: 'object',
+      title: '',
+      properties: {
+        [getEntityAddressKey(0, FORM, DATETIME_ADMINISTERED)]: {
+          type: 'string',
+          title: 'Intake Date',
+          format: 'date',
+          default: DateTime.fromISO(currentDateTime).toISODate()
```

Comment:
Why not just `DateTime.local().toISODate()`?

Reply:
hmm when i first made this form, i wanted the intake date and status date to be exactly the same so i saved `DateTime.local().toISO()` to a variable to be used for each, but it doesn't really make sense... could've just saved `DateTime.local()` to a variable, but also now, i handle saving the exact same datetime in the `onSubmit`.